cfme/tests/configure/test_tag.py

```diff
@@ -124,6 +126,40 @@ def test_map_tagging_crud(appliance, category, soft_assert):
                                           .format(map_tag_entity.label))
 
 
+@pytest.fixture
+def csv_tag_file(create_vm, category, tag):
+    temp_file = tempfile.NamedTemporaryFile(suffix=".csv")
+    csv_data = f'name,category,entry\n{create_vm.name},{category.display_name},{tag.display_name}'
+    with open(temp_file.name, "w") as file:
+        file.write(csv_data)
+    yield file.name
```

Comment:
Can this `yield` be moved into the context manager?

Reply:
I believe the file would still be open if I yielded inside of the context manager.  I want to yield the file name after the file is closed via the context manager. Am I misunderstanding something here?

Reply:
That way I don't need to close the file explicitly.

Reply:
Ah okay, makes sense -- thanks!